Document behaviour of SEQUENCES (#174)
clockworklabs/spacetime-docs: docs/appendix.md

# Appendix

## SEQUENCE

> For each table containing an `#[auto_inc]` column, a sequence number `generator` is automatically created behind the scenes, functioning similarly to `postgres` `SEQUENCE`.

Reviewer: ```suggestion
For each table containing an `#[auto_inc]` column, SpacetimeDB creates a sequence number `generator` behind the scenes, which functions similarly to `postgres`'s `SEQUENCE`.
```

Reviewer: Our style guide requires avoiding "passive voice"

> For each table containing an `#[auto_inc]` column, SpacetimeDB creates a sequence number `generator` behind the scenes, which functions similarly to `postgres`'s `SEQUENCE`.

Reviewer 2: ```suggestion
For each table containing an `#[auto_inc]` column, SpacetimeDB creates a sequence number generator behind the scenes, which functions similarly to `postgres`'s `SEQUENCE`.
```

"generator" is not a function or variable name here.
